docs/bootstrap_cces_azure.md

```diff
@@ -0,0 +1,50 @@
+## rubrik_bootstrap_cces_azure
+
+The rubrik_bootstrap_cces_azure resource will complete the bootstrap process for a Rubrik Cloud Cluster ES on Azure. You will also need to configure the Rubrik provider with the "username" and "password" set to blank strings.
+
+## Example Usage
+
+```hcl
+resource "rubrik_bootstrap_cces_azure" "example" {
+  cluster_name           = "tf-demo"
+  admin_email            = "tf@demo.com"
+  admin_password         = "RubrikTFDemo2019"
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High <!-- 07bb1a38-fdd0-4003-b578-783b2cbbda15 -->

Reply:
This is an example and not a real secret.